### PR TITLE
Skip the TestConcurrency for now

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by hickeng** [comment]
LGTM so long as there's an issue to re-enable these.
Also, hickeng cannot review tests...

**Comment by caglar10ur** [comment]
#1784 is created for that

File: lib/dns/cache_test.go [hunk 1]
```diff
@@ -159,6 +159,8 @@ func TestExpiration(t *testing.T) {
 
 // For best result run with -race
 func TestConcurrency(t *testing.T) {
+	t.Skipf("Caglar can't write tests...")
```
Comment (fdawg4l) [inline lib/dns/cache_test.go]: LMAO!!!

**Comment by fdawg4l** [comment]
LGTM

**Pull request merged**